### PR TITLE
Optimize the inlined strlen/memcpy/memset

### PR DESCRIPTION
**Pull request by pfusik**

### CODE REVIEW

**Comment by clbr** [comment]
The first one looks like it'd break the branch? bne only works on the A register.

**Comment by clbr** [comment]
Sorry, ldx does set the Z flag, nevermind.

File: src/cc65/stdfunc.c [hunk 1]
```diff
@@ -284,7 +284,7 @@ static void StdFunc_memcpy (FuncDesc* F attribute ((unused)), ExprDesc* Expr)
             Label = GetLocalLabel ();
 
             /* Generate memcpy code */
-            if (Arg3.Expr.IVal <= 127) {
+            if (Arg3.Expr.IVal <= 129) {
```
Comment (oliverschmidt) [inline src/cc65/stdfunc.c]: Please explain a bit why this can be changed from 127 to 129.

Comment (pfusik) [inline src/cc65/stdfunc.c]: These conditions are for loops like:

         ldy #IVal-1
    loop ...
         dey
         bpl loop

They will work for `ldy #128` and lower, hence IVal<=129.

Comment (oliverschmidt) [inline src/cc65/stdfunc.c]: Thanks for the explanation :-)

**Pull request merged**